yaml/pois.yaml

```diff
@@ -288,7 +288,7 @@ filters:
     min_zoom: 18
     output: {kind: toilets}
   - filter: {barrier: gate}
-    min_zoom: 15
+    min_zoom: {17 - COALESCE(mz_get_highway_level_point(osm_id), 0)}
```

Comment:
Should we return the **min_zoom** value in the function, instead of needing the `17 - ...` logic? What do we do for the trails logic?